### PR TITLE
Update working-with-pipelines.md

### PR DESCRIPTION
**Pull request by aadeshpa**
Answer update to this question in tls section @aadeshpa how do you get the ca.crt? can we add a pointer here?

### CODE REVIEW

File: publish/working-with-pipelines/working-with-pipelines.md [hunk 1]
```diff
@@ -210,7 +210,21 @@ status:
   internalRegistryHostname: image-registry.openshift-image-registry.svc:5000
 ```
 
-- Once you have the external route and the `ca.crt` for the internal image registry external route, use the steps in section, `Private image registries`.  @aadeshpa how do you get the ca.crt?  can we add a pointer here?
+- Once you have the external route setup you can find the ca.crt value for the external route as below from the cluster
```
Comment (jantley-ibm) [inline publish/working-with-pipelines/working-with-pipelines.md]: change to:
Once you have the external route set up, you can find the value for the external route in the `ca.crt` certificate.

File: publish/working-with-pipelines/working-with-pipelines.md [hunk 2]
```diff
@@ -210,7 +210,21 @@ status:
...
+oc get configmap  -n openshift-image-registry
+
+NAME                          DATA   AGE
+image-registry-certificates   2      7d
+serviceca                     1      7d
+trusted-ca                    1      7d
+[root@rugger-inf ~]# 
+
+```
+  From the above command results try to search the content of configmap `image-registry-certificates` and find relevant external route certificate.
```
Comment (jantley-ibm) [inline publish/working-with-pipelines/working-with-pipelines.md:222]: Change to:
Search the contents of the `image-registry-certificates` configmap to find the relevant external route certificate. Note the certificate value for the external route.

File: publish/working-with-pipelines/working-with-pipelines.md [hunk 3]
```diff
@@ -210,7 +210,21 @@ status:
...
+  
+- Once you get the certifiate value for internal image registry external route from previous step, use the steps in section, [Private image registries](https://github.com/kabanero-io/guides/blob/master/publish/working-with-pipelines/working-with-pipelines.md#private-image-registries) to create a configmap in `openshift-config` namespace with the `key` as your internal registry external route as `hostname` and value as the content of your external route certificate value.
```
Comment (jantley-ibm) [inline publish/working-with-pipelines/working-with-pipelines.md:224]: change to:
Use the steps in the [Private image registries](https://github.com/kabanero-io/guides/blob/master/publish/working-with-pipelines/working-with-pipelines.md#private-image-registries) section of the [Working with pipelines](https://github.com/kabanero-io/guides/blob/master/publish/working-with-pipelines/working-with-pipelines.md) guide to create a configmap in the `openshift-config` namespace. For the key, set `hostname` to your internal registry external route, and set `value` to the certificate value for the external route.

Comment (jantley-ibm) [reply publish/working-with-pipelines/working-with-pipelines.md:224]: Not sure I understood all of this correctly. Let me know if something I changed made it incorrect.

**Review by jantley-ibm — COMMENTED** [review]
Just a few edits, let me know if you have questions or if I misunderstood.

File: publish/working-with-pipelines/working-with-pipelines.md (hunk 1 above)
Comment (aadeshpa) [reply publish/working-with-pipelines/working-with-pipelines.md]: It should be something like this : Once you have the external route set up, you can find the ca certificate value for the external route hostname by below steps from the cluster.

File: publish/working-with-pipelines/working-with-pipelines.md (hunk 2 above)
Comment (aadeshpa) [reply publish/working-with-pipelines/working-with-pipelines.md:222]: looks good.

File: publish/working-with-pipelines/working-with-pipelines.md (hunk 3 above)
Comment (aadeshpa) [reply publish/working-with-pipelines/working-with-pipelines.md:224]: This looks good.

File: publish/working-with-pipelines/working-with-pipelines.md (hunk 1 above)
Comment (jantley-ibm) [reply publish/working-with-pipelines/working-with-pipelines.md]: Thanks for the clarification. For accessibility reasons (people using screen readers) we're not supposed to use directional terms like "above" and "below". So let's go with:

Once you have the external route set up, you can find the ca certificate value for the external route hostname by following these steps from the cluster.

Comment (aadeshpa) [reply publish/working-with-pipelines/working-with-pipelines.md]: sure, looks fine.

Comment (aadeshpa) [reply publish/working-with-pipelines/working-with-pipelines.md]: updated this in the same branch.

**Review by jantley-ibm — APPROVED** [review]
LGTM

**Pull request merged**